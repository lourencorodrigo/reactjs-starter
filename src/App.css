@import url("https://fonts.googleapis.com/css?family=Josefin+Sans:300,400");
.title {
  color: #2c7f94;
  font-family: 'Josefin Sans';
  font-weight: 400; }
