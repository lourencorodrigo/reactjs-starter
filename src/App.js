import React, { Component } from 'react';

import styles from './App.css';

class App extends Component {
  render() {
    const title = 'React';
    return (
      <h1 className={styles.title}>Welcome to {title}</h1>
    );
  }
}

export default App;
