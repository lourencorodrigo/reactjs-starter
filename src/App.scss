@import url('https://fonts.googleapis.com/css?family=Josefin+Sans:300,400');

.title {
  color: rgb(44, 127, 148);
  font-family: 'Josefin Sans';
  font-weight: 400;
}