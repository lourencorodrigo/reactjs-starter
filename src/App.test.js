import React from 'react';
const ReactTestRenderer = require('react-test-renderer');
import App from './App.js';

describe('Our first snapshot test', () => {
  it('Should compare the component with a snapshot', () => {
    const component = ReactTestRenderer.create(<App />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
