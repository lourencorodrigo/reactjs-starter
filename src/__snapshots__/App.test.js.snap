// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Our first snapshot test Should compare the component with a snapshot 1`] = `
<h1
  className={undefined}
>
  Welcome to 
  React
</h1>
`;
